extensions/ql-vscode/test/unit-tests/common/unzip.test.ts

```diff
@@ -0,0 +1,113 @@
+import { resolve } from "path";
+import {
+  excludeDirectories,
+  openZip,
+  openZipBuffer,
+  readZipEntries,
+} from "../../../src/common/unzip";
+
+const zipWithSingleFilePath = resolve(
+  __dirname,
+  "../../vscode-tests/no-workspace/data/archive-filesystem-provider-test/single_file.zip",
```

Comment:
Should we be moving these to a common location? It's currently in a directory for integration tests but is used in unit tests. Or should we perhaps have completely separate test data for these tests?

Reply:
I can add separate test data for these, but I'm somewhat hesistant to add binary files in a Git repository. However, since for `unzipToDirectory` we'll probably need to have separate test data anyway to test the `mode`, I can just add some test data now.